tests/unit_tests/test_all.py

```diff
@@ -0,0 +1,72 @@
+import json
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def all_process_code():
+    return load_process_code("all")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({"data": [False, None]}, False),
+        ({"data": [True, None]}, True),
+        ({"data": [False, None], "ignore_nodata": False}, False),
+        ({"data": [True, None], "ignore_nodata": False}, None),
+        ({"data": [True, False, True, False]}, False),
+        ({"data": [True, False]}, False),
+        ({"data": [True, True, True]}, True),
+        ({"data": [True]}, True),
+        ({"data": [None], "ignore_nodata": False}, None),
+        ({"data": []}, None),
+        ({"data": [], "ignore_nodata": False}, None),
+        ({"data": [True]}, True),
```

Comment:
Duplicate of line 23, so it can probably be deleted.

FFTR when removed.

Reply:
Eh, github has different workflow so every change has to be approved even if the PR was approved before. You can ping me so you won't have to wait for another approval.

Reply:
Deleted